### ISSUE TITLE
Websocket RPC support

### ISSUE THREAD

**Issue by 4lve** [metaplex-foundation/umi#110]
Would be cool if websocket rpc support was added, or atleast a setting in `umi.rpc.call`

**Comment by 4lve** [comment]
Found a temp fix, using 
```typescript
(umi.rpc as any).connection as Connection
```

Don't use this in production code tho as in the future the rpc is going to be rewritten.

**Comment by lorisleiva** [comment]
Yeah Umi current lacks a `RpcSubscriptionsInterface` but my plan is to add support for it when I update Umi to be compatible with the new web3.js.

Thanks for raising an issue though. I'll leave it open until there's an official solution for it.

**Comment by SS7896** [comment]
> Would be cool if websocket rpc support was added, or atleast a setting in `umi.rpc.call`
